data_steward/tools/create_combined_dataset.py

```diff
@@ -0,0 +1,235 @@
+                        dest='console_log',
+                        action='store_true',
+                        required=False,
+                        help='Log to the console as well as to a file.')
+
+    common_args, unknown_args = parser.parse_known_args(raw_args)
+    custom_args = clean_cdr._get_kwargs(unknown_args)
+    return common_args, custom_args
+
+
+def main(raw_args=None):
+    """
+    Clean an combined import.
+
+    Assumes you are passing arguments either via command line or a
+    list.
+    """
+    args, kwargs = parse_combined_args(raw_args)
+
+    pipeline_logging.configure(level=logging.INFO,
+                               add_console_handler=args.console_log)
+
+    # validate we've got all required data before continuing
+    cleaning_classes = clean_cdr.DATA_STAGE_RULES_MAPPING.get('combined')
+    clean_cdr.validate_custom_params(cleaning_classes, **kwargs)
+
+    # get credentials and create client
+    impersonation_creds = auth.get_impersonation_credentials(
+        args.run_as_email, CDR_SCOPES)
+
+    bq_client = BigQueryClient(args.curation_project_id,
+                               credentials=impersonation_creds)
+
+    # create staging, sandbox, and clean datasets with descriptions and labels
+    datasets = create_datasets(bq_client, args.combined_dataset,
+                               args.release_tag)
+
+    # copy raw data into staging dataset
+    copy_raw_combined_tables(bq_client, args.combined_dataset,
+                             datasets.get('staging'))
+
+    # clean the combined staging dataset
+    cleaning_args = [
+        '-p', args.curation_project_id, '-d',
+        datasets.get('staging', 'UNSET'), '-b',
+        datasets.get('sandbox', 'UNSET'), '--data_stage', 'combined',
+        "--cutoff_date", args.cutoff_date, '--validation_dataset_id',
+        args.validation_dataset_id, '--ehr_dataset_id',
+        args.unioned_ehr_dataset, '--api_project_id', args.api_project_id,
+        args.export_date, '--run_as', args.run_as_email, '-s'
+    ]
+
+    all_cleaning_args = add_kwargs_to_args(cleaning_args, kwargs)
+    clean_cdr.main(args=all_cleaning_args)
+
+    bq.build_and_copy_contents(bq_client, datasets.get('staging', 'UNSET'),
+                               datasets.get('clean', 'UNSET'))
+
+    # update sandbox description and labels
+    sandbox_dataset = bq_client.get_dataset(datasets.get(
+        'sandbox', 'UNSET'))  # Make an API request.
+    sandbox_dataset.description = (
+        f'Sandbox created for storing records affected by the cleaning '
+        f'rules applied to {datasets.get("clean")}')
+    sandbox_dataset.labels['phase'] = 'sandbox'
+    sandbox_dataset = bq_client.update_dataset(
+        sandbox_dataset, ["description"])  # Make an API request.
+
+    full_dataset_id = f'{sandbox_dataset.project}.{sandbox_dataset.dataset_id}'
+    LOGGER.info(
+        f'Updated dataset `{full_dataset_id}` with description `{sandbox_dataset.description}`'
+    )
```

Comment:
This could become
```suggestion
    LOGGER.info(
        f'Updated dataset `{sandbox_dataset.full_dataset_id}` with description `{sandbox_dataset.description}`'
    )
```